packages/renderer/src/lib/dialogs/QuickPickInput.svelte

```diff
@@ -74,7 +74,14 @@ const showQuickPickCallback = async (options?: QuickPickOptions) => {
     prompt = options.prompt;
   }
   mode = 'QuickPick';
-  quickPickItems = options.items.map(item => ({ value: item, checkbox: false }));
+  quickPickItems = options.items.map(item => {
+    // if type is QuickPickItem use label field for the display
+    if (item.label) {
```

Comment:
Wouldn't it be better to define a specific type for the QuickPickItem? Atm it's any. It seems to me that it can be string | {label: string, ...}

Reply:
@lstocchi I've amended the PR